runtime/queries/dart/indents.scm

```diff
@@ -12,7 +13,6 @@
   (arguments)
   (try_statement)
 ] @indent.begin
-
```

Comment:
Missing empty line?

Reply:
This seems to be gone in _every_ `indents.scm` after the `@indent.begin`...

Reply:
That doesn't make sense :thinking: Guess I messed up somewhere

Reply:
Well guess I was dumb :)
```
(program
  . (_)
  (_) @_comment
  .
  [
    (list)
    (grouping)
    (named_node)
    (anonymous_node)
  ] @format.prepend-newline
  (#not-has-type? @_comment comment))
```

This completely ignores the second node, all the time. Which happens to be what this pattern matches

Reply:
It's okay again now :). Sorry for the mistake

Reply:
https://github.com/nvim-treesitter/nvim-treesitter/compare/cfcc62d53a7c246493c3dd1fa80f22ed43b1e120..aad089b3adc0c481ffd3bc89148b850e83a05a87

This diff shows that all files were missing that separation of the first pattern. Got lost in the sauce

Reply:
Yup. Thanks for the fast fix!